### DISCUSSION TITLE
Why don't I get the selected prop to my NodeView

### DISCUSSION THREAD

**Discussion by mattiastofte** [ueberdosis/tiptap#2818] [General Questions]
Hi,
I've used the documentation as described [here](https://tiptap.dev/guide/node-views/react#selected). The problem is that I get all the other props, just not the 'selected' prop which I've defined as a boolean value in typescript. Why is this? I'm using React with NextJS.

**Comment by sereneinserenade** [comment]
if you want to use the node selection related things, you might wanna check out https://tiptap.dev/api/extensions/focus
